frontend/index.html

```diff
@@ -43,7 +43,7 @@ <h2>Settings</h2>
         <div id='paste-expiration-panel' class='paste-setting-subitem-panel'>
             <input list='expiration-choices' type='text' min='60' step='1' name='paste-expiration'
                    id='paste-expiration-input' placeholder='Expiration (secs)' value='7d'>
-            <label class='small-label' for='paste-expiration-input'>Delete your paste after a period of time. <br>Units: s (seconds), m (minutes), h (hours), d (days), M (months)</label>
+            <label class='small-label' for='paste-expiration-input'>Delete your paste after a period of time (leave blank for no expiry). <br>Units: s (seconds), m (minutes), h (hours), d (days), M (months), Y (years) <br> Max 68 years</label>
```

Comment:
`Max 68 years` could be omitted since I don't think the service can live so long and I prefer the label to be as concise as possible.